pytest_fixtures/component/provision_pxe.py

```diff
@@ -329,7 +329,7 @@ def pxeless_discovery_host(provisioning_host, module_discovery_sat, pxe_loader):
         job_template='configure-pxe-boot',
         target_host=provisioning_host.name,
         target_vlan_id=settings.provisioning.vlan_id,
-        target_vm_firmware=provisioning_host._broker_args['target_vm_firmware'],
+        target_vm_firmware=provisioning_host.target_vm_firmware,
```

Comment:
I think this should remain same as we've _broker_args attributes

Reply:
Updated